extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -369,6 +375,20 @@ function addLabelInputPairToContainer(view: azdata.ModelView, components: azdata
 }
 
 async function processField(context: FieldContext): Promise<void> {
+	context.fieldValidations = context.fieldInfo.validations?.map((validation => createValidation(
```

Comment:
This populates the fieldValidation objects for each fields based on the information from the fieldInfo